.gitignore

```diff
@@ -1 +1,3 @@
 build/
+*.pyc
+template/**/__pycache__
```

Comment:
Can we make this look like this:

https://github.com/AFCYBER-DREAM/piedpiper-flake8-faas/blob/master/.gitignore

We should try to keep the gitignore for all these functions the same